consultation_analyser/pipeline/backends/bedrock_llm_backend.py

```diff
@@ -0,0 +1,22 @@
+from django.conf import settings
+from langchain_aws import BedrockLLM
+
+from .langchain_llm_backend import LangchainLLMBackend
+
+
+class BedrockLLMBackend(LangchainLLMBackend):
+    def __init__(self):
+        llm = BedrockLLM(
+            # hardcoding this because the kwargs and model
+            # are coupled - can generalise later
+            model_id="mistral.mistral-large-2402-v1:0",
```

Comment:
We should parameterise `model_id`. We may be changing it in future based on our used cases.

Reply:
See comment! For now it's coupled to the model_kwargs which include references to `[INST]` etc. Can def change later